plugins/scenario/hu.bme.mit.gamma.scenario.language/src/hu/bme/mit/gamma/scenario/language/ScenarioLanguage.xtext

```diff
@@ -30,6 +30,7 @@ ScenarioDeclaration returns ScenarioModel::ScenarioDeclaration:
 	annotation+=Annotations*
 	'scenario' name=ID ('('(parameterDeclarations+=ParameterDeclaration)(',' parameterDeclarations+=ParameterDeclaration)*')')? 
 	(initialblock=InitialBlock)?
+	(variableDeclarations+=VariableDeclaration)*
```

Comment:
I believe the variable declarations should be put before the initial block, so the variables can also be used from there.